files/fr/web/api/webgl_api/tutorial/using_textures_in_webgl/index.md

```diff
@@ -11,7 +11,7 @@ Maintenant que notre programme peut faire tourner un cube 3D, appliquons lui une
 
 La première chose à faire est d'ajouter le code pour charger les textures. Dans notre cas, nous utiliserons une texture unique, appliquée à chacune des six faces de notre cube en rotation ; mais la même technique peut être utilisée un nombre quelconque de textures.
 
-> **Note :** il est important de noter que le chargement des textures suit les [règles inter-domaines](/fr-FR/docs/Web/HTTP/CORS)&nbsp;; donc vous pouvez seulement charger des textures depuis les sites pour lesquels votre contenu a l'approbation CORS. Voir les textures inter-domaines ci-dessous pour plus de détails.
```

Comment:
I'm not sure could I remove the space `&nbsp;;`, if could not, feel free tor restore it.